qiskit_experiments/framework/base_experiment.py

```diff
@@ -269,8 +269,7 @@ def run(
         # Generate and transpile circuits
         transpile_opts = copy.copy(experiment.transpile_options.__dict__)
         transpile_opts["initial_layout"] = list(experiment.physical_qubits)
-        circuits = transpile(experiment.circuits(), experiment.backend, **transpile_opts)
-        experiment._postprocess_transpiled_circuits(circuits, **run_options)
+        circuits = self._transpile(experiment.circuits(), experiment.backend, **transpile_opts)
```

Comment:
Could you please rename this to `transpiled_circuit` for readability?

Reply:
Do you mean `transpiled_circuits`? I think that name is confusing because it makes it seem like this is a version of `BaseExperiment.circuits` which just generates the transpiled versions instead. In reality, it takes the circuits as input and has the same function signature as `qiskit.transpile`. The analogy with `qiskit.transpile` is the reason I think the current name is the best. What do you think?

Reply:
Indeed this is nitpick and not exposed to external users. However for developers logical vs physical circuit is really important. If a developer reads this line
```python
jobs = experiment._run_jobs(circuits, **run_opts)
```
one may wonder `circuits` is physical or not (of course it is physical).